### PR TITLE
Run CIFuzz on pull requests

### PR DESCRIPTION
**Pull request by catenacyber**
Not to be merged before checking

Inspired by https://github.com/google/syzkaller/pull/1581

Plan is to solve https://github.com/google/gonids/pull/116#issuecomment-565261116

### CODE REVIEW

File: .github/workflows/fuzz.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    steps:
+    - name: Building and running fuzzers
+      uses: google/oss-fuzz/infra/cifuzz/actions@master
+      with:
+        project-name: 'gonids'
+        fuzz-time: 360
+        dry-run: True
+    - uses: actions/upload-artifact@v1
+      with:
+        name: fuzzer_parserule
```
Comment (Dor1s) [inline .github/workflows/fuzz.yml]: I think these are not fuzz target's name and path, these are for the artifacts created after fuzzing, so would be better to use the values from https://github.com/google/syzkaller/pull/1581/files //cc @leo-neat

**Comment by duanehoward** [comment]
Will wait to merge until check passes and @Dor1s is good with change.

File: .github/workflows/fuzz.yml (hunk 1 above)
Comment (Leo-Neat) [reply .github/workflows/fuzz.yml]: path: ./out/testcase should fix the failure.
Sorry for the confusion. https://github.com/google/oss-fuzz/pull/3342  will switch the upload artifact from just a single testcase file to as it is above, to a directory containing the testcase files and a file containing the crash stack trace. Is there any other information that you would like to be added to the upload @catenacyber ?

Comment (catenacyber) [reply .github/workflows/fuzz.yml]: >  Is there any other information that you would like to be added to the upload

I do not know what you mean here. I would like to run fuzzing as a regression test against pull requests with the oss-fuzz corpus.

**Comment by duanehoward** [comment]
Closing, was addressed in PR #137

**Pull request closed without merging**